modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -577,7 +577,7 @@ export const console: ConsoleNS = {
                             }
                         }
                     },
-                    placeholder: "Search by application name or clientId"
+                    placeholder: "Search applications by name or clientId or issuer"
```

Comment:
Is it appropriate to use 'or' many times in a sentence? Can we verify this sentence with the Docs team?

Suggestion: Search applications by name, clientId or issuer